Bottom: 371fec566621ddef971ae3ef78a260e31c42b2e0
Top:    7959d8aa37337978404ed6529cfb74a2df66473d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-04 21:39:08 +0000

Refresh of hid-gtk-gl-i-think-the-polygon

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 22c33f5..b26cde1 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1405,7 +1405,7 @@ GhidDrawMask (int side, BoxType * screen)
   polygon.BoundingBox = *screen;
   polygon.Flags = NoFlags ();
   SET_FLAG (FULLPOLYFLAG, &polygon);
-  gui->graphics->fill_pcb_polygon (out->fgGC, &polygon, screen);
+  hid_draw_fill_pcb_polygon (out->fgGC, &polygon, screen);
   poly_FreeContours (&polygon.NoHoles);
   ghid_set_alpha_mult (out->fgGC, 1.0);
